presto-docs/src/main/sphinx/release/release-0.214.rst

```diff
@@ -0,0 +1,64 @@
+=============
+Release 0.214
+=============
+
+General Changes
+---------------
+
+* Fix history leak in coordinator for failed or canceled queries.
+* Fix memory leak related to query tracking in coordinator that was introduced
+  in 0.213.
+* Fix planning failures when lambdas are used in join filter expression.
+* Send proper error response to client for certain types of errors during query
+  creation.
+* Improve error message when comparator provided to :func:`array_sort` function
+  does not satisfy the required contract.
+* Improve performance for lookup operations on map data types.
+* Improve planning and query times for queries with ``TINYINT``, ``SMALLINT``
+  and ``VARBINARY`` literals.
+* Maintain order of predicates and join tables in the query plan.
+* Make distributed ``ORDER BY`` and aggregation unspill progress on early yield
+  signals.
+* Make top-N row number optimization work in some cases when columns are pruned.
+* Add session feature toggle ``optimizer.optimize-to-n-row-number`` for top-N
+  row number optimization (enabled by default)
+* Add :func:`ngrams` function to generate the N-grams from an array.
+* Add :ref:`qdigest <qdigest_type>` type and associated :doc:`qdigest`
+  functions.
+* Add functionality to delay query execution until a minimum number of workers
+  nodes is available. The minimum number of workers can be set with the
+  ``query-manager.required-workers`` configuration property, and the max wait
+  time can ``query-manager.required-workers-max-wait`` configuration property.
+* Remove experimental pre-allocated memory system, and the related configuration
+  property ``experimental.preallocate-memory-threshold``.
+
+Security Changes
+----------------
+
+ * Add ``security.refresh-period`` configuration property to file-based access
+   controllers that allows to reload their configuration.
```

Comment:
This sounds strange

Reply:
Is it better?

>  * Add ``security.refresh-period`` configuration property to allow to reload the
   configuration of file-based access controllers.

The related change is e748ffe.